packages/admin/admin/src/table/filterbar/filterBarPopoverFilter/FilterBarPopoverFilter.tsx

```diff
@@ -9,30 +10,132 @@ import { messages } from "../../../messages";
 import { dirtyFieldsCount } from "../dirtyFieldsCount";
 import { FilterBarActiveFilterBadgeProps } from "../filterBarActiveFilterBadge/FilterBarActiveFilterBadge";
 import { FilterBarButton, FilterBarButtonProps } from "../filterBarButton/FilterBarButton";
-import { FilterBarPopoverFilterClassKey, styles } from "./FilterBarPopoverFilter.styles";
 
 /**
  * @deprecated Use MUI X Data Grid in combination with `useDataGridRemote` instead.
  */
-export interface FilterBarPopoverFilterProps {
+
+export type FilterBarPopoverFilterClassKey =
+    | "root"
+    | "fieldBarWrapper"
+    | "popoverContentContainer"
+    | "paper"
+    | "buttonsContainer"
+    | "contentContainer";
+
+const Root = styled("div", {
+    name: "CometAdminFilterBarPopoverFilter",
+    slot: "root",
+    overridesResolver(_, styles) {
+        return [styles.root];
+    },
+})(
+    ({ theme }) => css`
+        background-color: ${theme.palette.common.white};
+        position: relative;
+        margin-bottom: 10px;
+        border-radius: 2px;
+        margin-right: 6px;
+    `,
+);
+
+const FieldBarWrapper = styled("div", {
+    name: "CometAdminFilterBarPopoverFilter",
+    slot: "fieldBarWrapper",
+    overridesResolver(_, styles) {
+        return [styles.fieldBarWrapper];
+    },
+})(
+    css`
+        position: relative;
+    `,
+);
+
+const PopoverContentContainer = styled("div", {
+    name: "CometAdminFilterBarPopoverFilter",
+    slot: "popoverContentContainer",
+    overridesResolver(_, styles) {
+        return [styles.popoverContentContainer];
+    },
+})(
+    css`
+        min-width: 300px;
+    `,
+);
+
+const StyledPopover = styled(Popover, {
+    name: "CometAdminFilterBarPopoverFilter",
+    slot: "paper",
+    overridesResolver(_, styles) {
+        return [styles.tabs];
+    },
+})(css`
+    margin-left: -1px;
+    margin-top: 2px;
+`);
+
+const ContentContainer = styled("div", {
+    name: "CometAdminFilterBarPopoverFilter",
+    slot: "contentContainer",
+    overridesResolver(_, styles) {
+        return [styles.contentContainer];
+    },
+})(
+    css`
+        box-sizing: border-box;
+        padding: 20px;
+        margin-bottom: 0;
+    `,
+);
```

Comment:
This is a new Element, that didn't exist before... those styles should be added to `CometAdminFormFieldContainer-root` inside the `PopoverContentContainer` slot.